fplll/householder.cpp

```diff
@@ -0,0 +1,590 @@
+/* Copyright (C) 2005-2008 Damien Stehle.
+   Copyright (C) 2007 David Cade.
+   Copyright (C) 2011 Xavier Pujol.
```

Comment:
You are missing here.

Reply:
Yes. In fact, I removed these lines from all the other files I have introduced. I am not really I big fan of putting my name in the beginning of the file (in fact, I am not sure that the naming convention in fplll is respected). I understand that this can be an issue for changing the license or other related things, then I may follow the rules for these reasons. What do you propose?

Reply:
I don't think you can remove them as these are legal statements. I'm not a lawyer, though. I agree that we do have copyright information in the form of git commits but I think it's still a good idea to claim copyright when having done something major. Thus, I'd propose that you add yourself. Indeed, if we wanted to change the license we'd have to reach out to all copyright holders.

Reply:
Since I have introduced this file (I just brainless copy the header of `gso.cpp`), this three first lines can maybe just be dropped, since they were not written by any of them (at least, I take the idea of using `row_expo` in `gso.cpp`). But I am also not a lawyer, and not aware about how we should proceed. I will follow your advice.

Reply:
Agreed, these lines should go here. Taking an idea from somewhere is not the same as copyright which about the literal code, not an idea in the code.

Reply:
Add copyright information with commit 94aba9b.